updater/lib/dependabot/setup.rb

```diff
@@ -39,7 +41,8 @@
     swift
   )}x
 
-  config.processors += [ExceptionSanitizer]
+  config.before_send = ->(event, hint) { Dependabot::Sentry::Processor.process_chain(event, hint) }
+  config.propagate_traces = false
```

Comment:
This prevents Sentry adding `Sentry-Trace` and `Baggage` headers by default to all outgoing requests. It's designed to make it easier to do distributed tracing between microservices or frontend/backend. As we don't have that architecture, and it currently breaks our smoke test cache, I've chosen to disable it by default.

If we want to get this distributed trace information, we'll need to ignore these headers in our updater proxy.

References:
- https://github.com/getsentry/sentry-ruby/blob/7666c94f1f2671e4c42cd8c4a6991e3ce7da56f9/sentry-ruby/lib/sentry/configuration.rb#L189-L191
- https://www.w3.org/TR/baggage/